CHANGELOG.md

```diff
@@ -11,6 +11,13 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Added the `updateTrustedDomain` GraphQL API, allowing users to modify a
   trusted domain.
+- Added `IpAddress` GraphQL custom scalar for IP addresses.
+  - Applied it to the GraphQL APIs `ipLocation`, `ipLocationList`,
+    `insertAccount`, `updateAccount`, `insertSamplingPolicy`, and
+    `updateSamplingPolicy`.
+  - The API returns the following error message when a value cannot be parsed as
+    an `IpAddr`: `Failed to parse "IpAddress": Invalid IP address: abc (occurred
+while parsing "[IpAddress!]")`, where "abc" is the input.
```

Comment:
When a quoted message overflows like this, we can use backticks to create a block instead.

 ```
  - The API returns the following error message when a value cannot be parsed as
    an `IpAddr` (e.g., when "abc" is given):
    ```
    Failed to parse "IpAddress": Invalid IP address: abc (occurred while parsing
    "[IpAddress!]")
    ```
```

Reply:
There was no error in CI, so I thought it was an acceptable phrase.
We will apply it according to your suggestion.
Thank you.